File: backend/app/utils/environment.py
import os


def getEnv(key: str):
    return os.getenv(key)


class Env:
    DATABASE_URL = getEnv("DATABASE_URL")
    DATABASE_DOCKER_URL = getEnv("DATABASE_DOCKER_URL")
    SECRET_TOKEN = getEnv("SECRET_TOKEN")
    SECRET_ANSWER = getEnv("SECRET_ANSWER")
    EMAIL_ADDRESS = getEnv("SUPPORT_EMAIL_ADDRESS")
    EMAIL_PASSWORD = getEnv("SUPPORT_EMAIL_PASSWORD")
    FRONTEND_URL = getEnv("FRONTEND_URL")
    ENDPOINT_URL = getEnv("ENDPOINT_URL")
    MINIO_ACCESS_KEY = getEnv("MINIO_ACCESS_KEY") or ""
    MINIO_SECRET_KEY = getEnv("MINIO_SECRET_KEY") or ""
    MINIO_URL = getEnv("MINIO_URL") or ""

File: backend/src/utils/environment.ts
export const Env = {
  PORT: process.env.PORT,
  DATABASE_URL: process.env.DATABASE_URL,
  SECRET_TOKEN: process.env.SECRET_TOKEN ?? "",
  FRONTEND_URL: process.env.FRONTEND_URL ?? "",
  MINIO_ENDPOINT: process.env.MINIO_URL ?? "",
  MINIO_ACCESS_KEY: process.env.MINIO_ACCESS_KEY ?? "",
  MINIO_SECRET_KEY: process.env.MINIO_SECRET_KEY ?? "",
};
